entrypoint.sh

```diff
@@ -3,14 +3,17 @@
 TEST_SUITE_NAME=$1
 ENABLE_DEBUG=$2
 CLUSTER_NAME=$3
-
+TESTKUBE_VERSION=$4
```

Comment:
New envar According to the installation file: https://kubeshop.github.io/testkube/install.sh, it gives us the option to specify the version we need.